tests/flux/test_flux_dev_pulid.py

```diff
@@ -0,0 +1,43 @@
+import pytest
+import torch
+from types import MethodType
+import numpy as np
+import torch.nn.functional as F
+from diffusers.utils import load_image
+
+from nunchaku.pipeline.pipeline_flux_pulid import PuLIDFluxPipeline
+from nunchaku.utils import is_turing
+from nunchaku import NunchakuFluxTransformer2dModel
+from nunchaku.models.pulid.pulid_forward import forward
+from nunchaku.models.pulid.utils import resize_numpy_image_long
+
+
+@pytest.mark.skipif(is_turing(), reason="Skip tests due to using Turing GPUs")
+def test_flux_dev_pulid():
+    transformer = NunchakuFluxTransformer2dModel.from_pretrained("mit-han-lab/svdq-int4-flux.1-dev")
```

Comment:
do not hardcore this to `int4`. Please set it like https://github.com/mit-han-lab/nunchaku/blob/c1d6fc8406c26307e4104a1b383d57c7b62640b1/examples/flux.1-dev.py#L7-8